processor/filterprocessor/metric_filter.go

```diff
@@ -0,0 +1,110 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package filterprocessor
+
+import (
+	"go.uber.org/zap"
+
+	"go.opentelemetry.io/collector/consumer/pdata"
+	"go.opentelemetry.io/collector/internal/processor/filtermetric"
+	"go.opentelemetry.io/collector/processor/processorhelper"
+)
+
+type MetricsFilterer struct {
```

Comment:
What do you think about changing this name (and the constructor) to just MetricsFilter?

Reply:
There was a `MetricFilter` struct in `filterprocessor`, that's why I named it MetricsFilterer initially. Updated `MetricsFilterer` to `MetricsFilter` and `MetricFilters` to `MetricsFilterConfig`.